src/components/ToDoItem.vue

```diff
@@ -1,3 +1,87 @@
-<template></template>
+<template>
+  <div
+    class="relative sm:mx-auto mx-4 flex sm:flex-col space-y-8 justify-center rounded-2xl bg-white border-black border-2 p-[18px] m-[50px] sm:w-[610px] w-80 flex-row"
+  >
+    <div class="w-full flex flex-row">
+      <div class="m-auto mr-5">
+        <div
+          class="p-2 border-black border-[3px] rounded-full w-6 h-6 flex-row lg:hidden"
+        ></div>
+      </div>
+      <div class="flex flex-col w-full items-center">
+        <p
+          class="w-full placeholder-black sm:text-5xl text-lg flex font-bold text-area lg:flex mt-5"
+        >
+          Title
+          {{ toDo.title }}
+        </p>
+        <div
+          class="flex w-full items-center sm:h-[12px] h-[5px] lg:flex hidden"
+        >
+          <img :src="dateIcon" class="lg:flex hidden mr-1 h-3 w-3" />
+          {{ newDate() }}
+        </div>
 
-<script></script>
+        <div class="flex w-full sm:h-[12px] lg:hidden text-gray-500">
+          <img :src="dateIcon" class="mr-1 h-3 w-3 hidden" />
+          {{ newDate() }}
+        </div>
+      </div>
+
+      <div class="flex">
+        <div
+          class="sm:invisible visible rounded-full h-2.5 w-2.5 my-9"
+          :class="{
+            'bg-green-500': 'Low' === localTodo.priority,
```

Comment:
Please create variables for these boolean variables, and use it here. Try not to use logic in the template.